crates/typst-library/src/compute/foundations.rs

```diff
@@ -1,4 +1,4 @@
-use typst::eval::EvalMode;
+use typst::{diag::SourceDiagnostic, eval::EvalMode};
```

Comment:
we generally don't use the nested import feature

Reply:
Removed